### PR TITLE
RSS: Episode 4

### PR DESCRIPTION
**Pull request by artem-zinnatullin**

### CODE REVIEW

File: feed.rss [hunk 1]
```diff
@@ -200,5 +200,58 @@
...
+			
+			<p>Chris Lacy <a href="https://twitter.com/chrismlacy">@chrismlacy</a></p>
+			
+			<p>Apps published by Chris Lacy on <a href="
+			https://play.google.com/store/apps/developer?id=Chris+Lacy">Google Play Store</a>:
+			 - <a href="https://play.google.com/store/apps/details?id=com.actionlauncher.playstore">Action Launcher 3</a>
+			 - <a href="https://play.google.com/store/apps/details?id=com.chrislacy.actionlauncher.pro">Action Launcher 2</a>
+			 - <a href="https://play.google.com/store/apps/details?id=com.tweetlanes.android">Tweet Lanes</a></p>
+			
+			<p>#### Hosts:</p>
```
Comment (sockeqwe) [inline feed.rss]: ####

File: feed.rss [hunk 2]
```diff
@@ -200,5 +200,58 @@
...
+				<li>Play Store, in app purchases and subscriptions</li>
+			</ul>
+			
+			<h4>Guest:</h4>
+			
+			<p>Chris Lacy <a href="https://twitter.com/chrismlacy">@chrismlacy</a></p>
+			
+			<p>Apps published by Chris Lacy on <a href="
+			https://play.google.com/store/apps/developer?id=Chris+Lacy">Google Play Store</a>:
+			 - <a href="https://play.google.com/store/apps/details?id=com.actionlauncher.playstore">Action Launcher 3</a>
```
Comment (sockeqwe) [inline feed.rss]: Shouldn't that be a `<ul>` ?

**Comment by artem-zinnatullin** [comment]
Fixed, thanks!

**Comment by artem-zinnatullin** [comment]
rePTAL

**Comment by artem-zinnatullin** [comment]
Decided to merge :)

**Comment by sockeqwe** [comment]
👍 
sorry for late response, wasn't online!

**Pull request merged**